app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(22, 4)
+    assert sum(result) == 22
```

Comment:
```suggestion
    assert sum(split_integer(22, 4)) == 22
```
no need to create variable, if it used only once